torchvision/prototype/transforms/_deprecated.py

```diff
@@ -0,0 +1,40 @@
+from typing import Any, Dict, Optional
+
+import numpy as np
+import PIL.Image
+from torchvision.prototype import features
+from torchvision.prototype.transforms import Transform
+from torchvision.transforms import functional as _F
+
+from ._utils import is_simple_tensor
+
+
+# TODO: add deprecation warning
```

Comment:
Is this going to happen later?

Reply:
Let's add the warnings to avoid having unfinished TODOs and we can always revise. This is a proposal after all, not necessarily the end result.